browser/src/shared/repo/backend.tsx

```diff
@@ -147,7 +147,12 @@ export const fetchBlobContentLines = memoizeObservable(
                 if (errors) {
                     if (errors.length === 1) {
                         const error = errors[0]
-                        const isFileContent = error.path.join('.') === 'repository.commit.file.content'
+                        const errorPath = error.path.join('.')
+
+                        // Originally this checked only for 'repository.commit.file.content'.
+                        // But if a file doesn't exist, the error path is 'repository.commit.file'
+                        const isFileContent =
+                            errorPath === 'repository.commit.file.content' || errorPath === 'repository.commit.file'
```

Comment:
Nice find! Given you verified this, I would lean towards not carrying the old, wrong check (that was probably valid _at some point_, but not anymore) around

cc @lguychard in case you have any concerns with this
```suggestion
                        const isFileContent = errorPath === 'repository.commit.file'
```

Reply:
I wasn't able to determine with confidence if the old value was still possible or not

Reply:
Since this is _only_ used to check if the error is "file does not exist", and that case is represented now with an error on the `file` field (as it should be, thinking about it), the only scenario I can see is old Sourcegraph instances in case this changed recently (since it apparently changed at _some_ point). I'd personally be willing to take a bet here against that, for the sake of getting rid of old cruft.
Just to be clear this doesn't block merging though.